<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Process_Data</class>
 <widget class="QMainWindow" name="Process_Data">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>682</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Process DB</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>312</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>312</width>
      <height>180</height>
     </size>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#qt_calendar_navigationbar{

	background-color: rgb(217, 16, 76);
}

#qt_calendar_prevmonth,
#qt_calendar_nextmonth{
border: none;
qproperty-icon: none;

min-width:13 px;
max-width:13 px;
min-height:13 px;
max-height: 13 px;

border-radius: 5px;

background-color: transparent;
padding: 5px

}

#qt_calendar_prevmonth{
	margin-right: 5px;
	image: url(:/Ico_Calendar/ico/Right.png);
}
#qt_calendar_nextmonth{
	margin-left: 5px;
	image: url(:/Ico_Calendar/ico/Left.png);
}

#qt_calendar_prevmonth:hover,
#qt_calendar_nextmonth:hover{
	background-color: rgb(0, 0, 0);
}

#qt_calendar_prevmonth:pressed,
#qt_calendar_nextmonth:pressed{
	background-color: rgb(0, 0, 0);
}

#qt_calendar_calendarview{

	border: 1px solid #d9104c;
	border-top: 0px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
    font-size: 10px; 
	padding: 2px;
    margin: 1px;

}

#qt_calendar_calendarview::item:hover{
	border-radius: 5px;
	color: rgb(255, 255, 255);
	background-color: #d9104c;
}

#qt_calendar_calendarview::item:selected{
	border-radius: 5px;
	background-color: #d9104c;
}
</string>
    </property>
    <property name="minimumDate">
     <date>
      <year>2024</year>
      <month>2</month>
      <day>1</day>
     </date>
    </property>
    <property name="maximumDate">
     <date>
      <year>2025</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="dateEditAcceptDelay">
     <number>1500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_Origins">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Selección de Origenes (Servicio)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Benefits">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(217, 16, 76);*/
font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Beneficios Aplicados</string>
    </property>
   </widget>
   <widget class="Line" name="line_Horizontal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>282</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_Brands">
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_0">
       <property name="text">
        <string>Marca 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_30">
       <property name="text">
        <string>Marca 30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_60">
       <property name="text">
        <string>Marca 60</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_90">
       <property name="text">
        <string>Marca 90</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_120">
       <property name="text">
        <string>Marca 120</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_brand_150">
       <property name="text">
        <string>Marca 150</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_180">
       <property name="text">
        <string>Marca 180</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_210">
       <property name="text">
        <string>Marca 210</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_Potencial">
       <property name="text">
        <string>Potencial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_Prechurn">
       <property name="text">
        <string>Prechurn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Brand_Castigo">
       <property name="text">
        <string>Castigo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>73</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_Origins">
     <item>
      <widget class="QCheckBox" name="checkBox_Origin_ASCARD">
       <property name="text">
        <string>ASCARD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Origin_BSCS">
       <property name="text">
        <string>BSCS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Origin_RR">
       <property name="text">
        <string>RR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Origin_SGA">
       <property name="text">
        <string>SGA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>555</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_Selections_Brands_FLP">
     <item>
      <widget class="QLabel" name="label_Brands">
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
       </property>
       <property name="text">
        <string>Selección de Marcas (Cartera)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_FLP">
       <item>
        <widget class="QLabel" name="label_Dates_FLP">
         <property name="styleSheet">
          <string notr="true">font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
         </property>
         <property name="text">
          <string>Selección de FLP (fecha límite de pago)         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_ALL_DATES_FLP">
         <property name="text">
          <string>Todas</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Process">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-radius:8px;
	color: rgb(0, 0, 0);
	background-color: rgb(209, 209, 209);
}
QPushButton:Hover
{
	color: rgb(255, 255, 255);
	background-color: rgb(217, 16, 76);
}
QPushButton:pressed
{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Benefits">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>400</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>10</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>--- Seleccione opción</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Con Descuento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sin Descuento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Todo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_LOGO">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>109</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/ico/Logo_Label.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_Vertical_Tittle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Contact_Min">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(217, 16, 76);*/
font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Números de Contacto</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Min_Contact">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>--- Seleccione opción</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Celular</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fijo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Todos</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Graphic">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-radius:8px;
	color: rgb(0, 0, 0);
	background-color: rgb(209, 209, 209);
}
QPushButton:Hover
{
	color: rgb(255, 255, 255);
	background-color: rgb(217, 16, 76);
}
QPushButton:pressed
{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="text">
     <string>Graficar</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Selection_Process">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 16, 76);
font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="text">
     <string>Proceso DataMarshall</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Selected_Process">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light</family>
      <pointsize>10</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;Bahnschrift Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>--- Seleccione opción</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BOT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EMAIL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IVR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SMS</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Mod_Init_CTA_Min">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Leelawadee UI&quot;;</string>
    </property>
    <property name="text">
     <string>Deuda Mínima</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mod_Init_CTA_Max">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Leelawadee UI&quot;;</string>
    </property>
    <property name="text">
     <string>Deuda Máxima</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mod_Init_Min">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>550</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mod_Init_Max">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>550</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mod_Init_CTA_">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Microsoft Yi Baiti&quot;;
color: rgb(163, 163, 163);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&gt;= $$ &lt;=</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Mod_Init_CTA_Group">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>489</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Bahnschrift SemiBold&quot;;</string>
    </property>
    <property name="title">
     <string>Filtro de Saldos</string>
    </property>
   </widget>
   <widget class="Line" name="line_Vertical_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Registers">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>620</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 16, 76);
font: 87 9pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>TOTAL DE REGISTROS:</string>
    </property>
   </widget>
   <widget class="Line" name="line_Vertical_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Registers_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>620</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(217, 16, 76);
font: 87 10pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>trial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Version_Control">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(162, 162, 162);
font: italic 10pt &quot;MS Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Proyecto de Automatización - V. 1.08.661</string>
    </property>
   </widget>
   <zorder>Mod_Init_CTA_Group</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>label_Origins</zorder>
   <zorder>label_Benefits</zorder>
   <zorder>line_Horizontal</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton_Process</zorder>
   <zorder>comboBox_Benefits</zorder>
   <zorder>label_LOGO</zorder>
   <zorder>line_Vertical_Tittle</zorder>
   <zorder>label_Contact_Min</zorder>
   <zorder>comboBox_Min_Contact</zorder>
   <zorder>pushButton_Graphic</zorder>
   <zorder>label_Selection_Process</zorder>
   <zorder>comboBox_Selected_Process</zorder>
   <zorder>label_Mod_Init_CTA_Min</zorder>
   <zorder>label_Mod_Init_CTA_Max</zorder>
   <zorder>lineEdit_Mod_Init_Min</zorder>
   <zorder>lineEdit_Mod_Init_Max</zorder>
   <zorder>label_Mod_Init_CTA_</zorder>
   <zorder>line_Vertical_2</zorder>
   <zorder>label_Total_Registers</zorder>
   <zorder>line_Vertical_3</zorder>
   <zorder>label_Total_Registers_2</zorder>
   <zorder>label_Version_Control</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="action_Files_Soported"/>
    <addaction name="action_Guide_2"/>
   </widget>
   <widget class="QMenu" name="menu_Creditos">
    <property name="title">
     <string>Créditos</string>
    </property>
    <addaction name="action_Developers"/>
   </widget>
   <widget class="QMenu" name="menu_Export">
    <property name="title">
     <string>Exportación</string>
    </property>
    <addaction name="action_Lists"/>
    <addaction name="action_Pictionary"/>
    <addaction name="action_Min_Corp"/>
   </widget>
   <addaction name="menu_Help"/>
   <addaction name="menu_Creditos"/>
   <addaction name="menu_Export"/>
  </widget>
  <action name="action_Guide">
   <property name="text">
    <string>Guía</string>
   </property>
  </action>
  <action name="action_HELP">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="action_Developers">
   <property name="text">
    <string>Desarrolladores</string>
   </property>
  </action>
  <action name="action_Guide_2">
   <property name="text">
    <string>Guía de uso</string>
   </property>
  </action>
  <action name="action_Files_Soported">
   <property name="text">
    <string>Archivos soportados</string>
   </property>
  </action>
  <action name="action_Lists">
   <property name="text">
    <string>Listas de referencia</string>
   </property>
  </action>
  <action name="action_Pictionary">
   <property name="text">
    <string>Diccionarios incluidos</string>
   </property>
  </action>
  <action name="action_Min_Corp">
   <property name="text">
    <string>Números espía (Corp)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/Picture.qrc"/>
 </resources>
 <connections/>
</ui>
